static/app/views/explore/logs/constants.tsx

```diff
@@ -29,7 +29,9 @@ const AlwaysHiddenLogFields: OurLogFieldKey[] = [
   OurLogKnownFieldKey.ID,
   OurLogKnownFieldKey.ORGANIZATION_ID,
   OurLogKnownFieldKey.ITEM_TYPE,
-  OurLogKnownFieldKey.PROJECT,
+  'project',
+  'project.id',
+  'project_id', // these are all aliases that might show up
```

Comment:
I think we may want to keep at least one of the project fields in the case of multi project search you can tell which project it is